src/metabase/api/user.clj

```diff
@@ -147,7 +148,16 @@
                                                         :permissions_group_membership
                                                         [:= :core_user.id :permissions_group_membership.user_id])
     (some? group_ids)                                  (sql.helpers/where
-                                                        [:in :permissions_group_membership.group_id group_ids])))
+                                                        [:in :permissions_group_membership.group_id group_ids])
+    (some? mw.offset-paging/*limit*)                   (sql.helpers/limit mw.offset-paging/*limit*)
+    (some? mw.offset-paging/*offset*)                  (sql.helpers/offset mw.offset-paging/*offset*)
+    true                                               (sql.helpers/order-by
+                                                        [:%lower.last_name :asc] [:%lower.first_name :asc])))
+
+(defn- count-clauses
+  "Given a clause, return the clause used to count."
```

Comment:
The docstring and name of this function didn't really make me understand what this was doing by itself. It's tied to how it's used in a as an argument to a `t2/count` expression, and requires that it's argument is a result from the `user-clauses` function, so it's usage is more narrow than its name suggests. Perhaps a more appropriate name would be `filter-clauses-without-paging`.